src/features/subscriptions/types.ts

```diff
@@ -0,0 +1,43 @@
+export enum BillingCycleID {
+  Monthly = 'month',
+  Annually = 'year',
+}
+
+export enum PlanStyle {
+  basic = 'basic',
```

Comment:
Two different naming convention for members of two enums :)

Tho, we don't really use enums, maybe let's types instead. 
'basic' | 'premium'